scholly/base/src/components/App.js

```diff
@@ -6,6 +6,11 @@ import { Switch, Route } from 'react-router-dom';
 import Header from './layout/Header.js';
 import { getRoutes } from './router/routes.js';
 import './centred.css';
+import './baseColours.css';
+import './darkColours.css';
+//import './lightColours.css';
+//import './thirdTheme.css';
+//import './fourthTheme.css';
```

Comment:
Please remove commented-out lines. We are planning to upgrade the SA for scholly base as well and this will throw a check.

Reply:
If you'd like to document that these are the available css stylesheets, I think it's better done over a comment or in the file docs at the top.

Reply:
@aksh0001 just resolving this because we've decided on just 2 colours now so this should be all good. Dynamic switching will be implemented after PR 226 is merged.